Documentation/SupportedCompilers.md

```diff
@@ -0,0 +1,21 @@
+ITK requires a compiler with C++11 support.
+
+# Visual Studio
+* MSVC 12 (VS2013) and earlier: **NOT supported**
+* MSVC 14 (VS2015): fully supported
+* MSVC 15 (VS2017): fully supported
+* MSVC 16 (VS2019): supported with some [caveats](https://discourse.itk.org/t/has-anyone-tried-to-build-visual-studio-c-2019/1561/6?u=dzenanz)
```

Comment:
This does not appear to be consistent with this:
https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering

Also please keep in mind a given Visual Studio can have multiple tool sets installed now.